teamcity: failed tests on release-2.0: lint/TestLint

The following tests appear to have failed:

[#593461](https://teamcity.cockroachdb.com/viewLog.html?buildId=593461):


```
--- FAIL: lint/TestLint (24.850s)

--- FAIL: lint/TestLint: TestLint/TestGolint (29.950s)
lint_test.go:952: sql/sqlbase/errors.go:236:24: should omit type error from declaration of var queryCanceledError; it will be inferred from the right-hand side
------- Stdout: -------
=== PAUSE TestLint/TestGolint
--- FAIL: lint/TestLint (24.850s)

--- FAIL: lint/TestLint: TestLint/TestGolint (29.950s)
lint_test.go:952: sql/sqlbase/errors.go:236:24: should omit type error from declaration of var queryCanceledError; it will be inferred from the right-hand side
------- Stdout: -------
=== PAUSE TestLint/TestGolint

```

Please assign, take a look and update the issue accordingly.

--- a/pkg/sql/sqlbase/errors.go
+++ b/pkg/sql/sqlbase/errors.go
@@ -233,7 +233,7 @@ func NewStatementCompletionUnknownError(err error) error {
 	return pgerror.NewErrorf(pgerror.CodeStatementCompletionUnknownError, err.Error())
 }
 
-var queryCanceledError error = pgerror.NewErrorf(
+var queryCanceledError = pgerror.NewErrorf(
 	pgerror.CodeQueryCanceledError, "query execution canceled")
 
 // NewQueryCanceledError creates a query cancellation error.

